### PR TITLE
Prepare merge of cva6/dev into master

### PR DESCRIPTION
**Pull request by zchamski**
Resolve conflicts in VPTOOL caused by commits that occurred on both `master` and `cva6/dev` but do not share a common history.  The end state of VPTOOL  is the one on branch `dev/cva6`.

Signed-off-by: Zbigniew Chamski <zbigniew.chamski@thalesgroup.com>

### CODE REVIEW

**Review by AyoubJalali — APPROVED** [review]

**Comment by AyoubJalali** [comment]
I reviewed files related to ISACOV & RVFI agent.

**Review by MikeOpenHWGroup — COMMENTED** [review]
LGTM, thanks to the CVA6 team for this.

The pull-request _looks_ big, but isn't.
- The largest contribution here (in terms of files), is `cva6/docs/VerifPlans` which has no direct impact on other cores.
- The update of `tools/vptool` is significant, but again will have no impact on other cores.

@silabs-mateilga, @silabs-robin, @silabs-krdosvik and @silabs-hfegran: one or more of you should look at the changes to `lib/uvm_agents/uvma_isacov` and `uvma_rvfi`.  I believe these all have been discussed previously, but a second review would be useful.

**Review by JeanRochCoulon — APPROVED** [review]
LGTM

File: lib/uvm_agents/uvma_isacov/uvma_isacov_tdefs.sv
Comment (silabs-robin) [inline lib/uvm_agents/uvma_isacov/uvma_isacov_tdefs.sv:559]: I remember this from discussions on gh issues. Look ok.

**Review by silabs-robin — APPROVED** [review]
Looks great. I have reviewed the changes to isacov and rvfi. It all seems familiar and sensible. Looking forward to trying out the fixes to isacov.

File: lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_mon.sv
Comment (AyoubJalali) [reply lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_mon.sv:41]: yeep

File: lib/uvm_agents/uvma_isacov/uvma_isacov_mon.sv
Comment (AyoubJalali) [reply lib/uvm_agents/uvma_isacov/uvma_isacov_mon.sv:216]: Yes no additional changes

**Pull request merged**
